source/lib.ts

```diff
@@ -411,7 +411,7 @@ const rateLimit = (
 			// Call the `onLimitReached` callback on the first request where client
 			// exceeds their rate limit
 			// TODO: `onLimitReached` is deprecated, this should be removed in v7.x
-			if (maxHits && totalHits === maxHits + 1) {
+			if (typeof maxHits === 'number' && totalHits === maxHits + 1) {
```

Comment:
Isn't `maxHits` always a number? (See [line 350](https://github.com/express-rate-limit/express-rate-limit/pull/380/files#diff-5c2516e9822679491492c832b9ace79b6945bdd891586e07b0232ebef1c7e0b1L350))

Reply:
Oh yea that's really silly of me lol

Reply:
Fixed in c7433c2